components/brave_wallet/browser/solana_instruction_builder.cc

```diff
@@ -292,8 +248,7 @@ std::optional<SolanaInstruction> Transfer(
   }
   size_t end = proof.proof.size() - proof.canopy_depth;
   for (size_t i = 0; i < end; ++i) {
-    account_metas.push_back(
-        SolanaAccountMeta(proof.proof[i], std::nullopt, false, false));
+    account_metas.emplace_back(proof.proof[i], std::nullopt, false, false);
```

Comment:
We should be getting a range of `proof.proof` with a span above, and then doing a for-range loop on it.

Reply:
https://github.com/brave/brave-core/pull/26431/commits/6809115eb5276f7e5f97305fc0e0189ade29d515